forge-gui/res/cardsfolder/d/dawnglow_infusion.txt

```diff
@@ -1,7 +1,9 @@
 Name:Dawnglow Infusion
 ManaCost:X GW
 Types:Sorcery
-A:SP$ GainLife | Cost$ X GW | LifeAmount$ X | ConditionManaSpent$ G | SubAbility$ WPaid | SpellDescription$ You gain X life if {G} was spent to cast this spell and X life if {W} was spent to cast this spell. (Do both if {G}{W} was spent.)
-SVar:WPaid:DB$ GainLife | LifeAmount$ X | ConditionManaSpent$ W
+A:SP$ GainLife | Cost$ X GW | LifeAmount$ Y | SpellDescription$ You gain X life if {G} was spent to cast this spell and X life if {W} was spent to cast this spell. (Do both if {G}{W} was spent.)
 SVar:X:Count$xPaid
+SVar:Y:SVar$W/Plus.G
+SVar:W:Count$Adamant_1.White.X.0
+SVar:G:Count$Adamant_1.Green.X.0
```

Comment:
@tool4ever wrong words

Adamant is affected by Color TextChange,
these use the Mana Symbols and should not be affected by TextChange

Reply:
Ugh right